flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js

```diff
@@ -2199,12 +2199,15 @@ var A320_Neo_UpperECAM;
         getValue(_engine) {
             const imbalance = SimVar.GetSimVarValue("L:A32NX_ENGINE_IMBALANCE", "number");
```

Comment:
What is the number format that comes out of this?
We're doing some very interesting substringing on it, so I'm curious.

Reply:
Its a coded "word"; I use it as a single LVar to give me all engine parameter imbalances (N2, FF and EGT). First two digits sets randomly and engine to be imbalanced (either 1 or 2). Then there are three sets of double digit values to set the imbalance for N2, FF and EGT. The FADEC wasm decodes that information.

Reply:
Okay. I'm not sure if you already did this, but please ensure this information is added to `docs/a320-simvars.md` (and you could consider pointing to there in a comment line in this code). Alternatively consider creating a JS class which reads and decodes this variable, such that we don't even need documentation 😃.

Reply:
Haven't added anything to the simvars.md since this is part of the experimental/ AP branch, but if needed I can start documenting the list of LVars I've created for the engine.

Regarding the JS class, I can try and add a decoding class in the future ... its just a parser and its being done in each parameter class already (N2, EGT and FF). But I can put that in the pipeline.

Reply:
Ah yes. The "file changes" view was hiding what came before it. I didn't see the parameter class. I think documentation is sufficient then!